cirq/ops/common_channels.py

```diff
@@ -548,7 +549,7 @@ def phase_damp(gamma: float) -> PhaseDampingChannel:
     return PhaseDampingChannel(gamma)
 
 
-@value.value_equality
+@value_equality
```

Comment:
This was really the only way to break the cycle? It's quite odd... we really don't want to have packages outside of a tree digging into the guts of that tree.

Reply:
Better now?

Reply:
Yes